lib/mintel/alerts/redis.libsonnet

```diff
@@ -0,0 +1,60 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'redis.alerts',
+        rules: [
+          {
+            alert: 'RedisMemoryUtilHigh',
+            expr: '(redis_memory_used_bytes / redis_memory_max_bytes  * 100 > 90 ) < 101',
```

Comment:
Do we really need the <101 ? 

i know we had it before in the rules ... just trying to remember why we need it since all we care about is that it has been > 90%

Reply:
Yeah it's a bit weird , upstream version does not have it, so I suspect @adamhf noted an issue in this area and changed it.

Reply:
should we remove it in the meantime ?

Reply:
I will do yep